src/routes.tsx

```diff
@@ -82,6 +83,11 @@ export const routes: RouteObject[] = [
                 element: <AccountTransactionsCard />,
                 loader: addressParamLoader,
               },
+              {
+                path: 'token-transfers',
```

Comment:
not sure where to fix this, but:
- https://0d99ad56.oasis-explorer.pages.dev/mainnet/sapphire/address/0x85495f44768ccbb584d9380Cc29149fDAA445F69 links to https://0d99ad56.oasis-explorer.pages.dev/mainnet/sapphire/address/0x85495f44768ccbb584d9380Cc29149fDAA445F69/token-transfers#transfers and it works
- https://0d99ad56.oasis-explorer.pages.dev/mainnet/sapphire/token/0x85495f44768ccbb584d9380Cc29149fDAA445F69 links to https://0d99ad56.oasis-explorer.pages.dev/mainnet/sapphire/token/0x85495f44768ccbb584d9380Cc29149fDAA445F69/token-transfers#transfers and that breaks

Reply:
Introducing the token dashboard page in #623 fixes this by re-defining what the `/token/` routes mean. There are no more broken links there.